tests/e2e/specs/empty-trash-restore-trashed-posts.test.js

```diff
@@ -0,0 +1,71 @@
+import {
+    visitAdminPage,
+    createNewPost,
+    trashAllPosts,
+    publishPost
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Empty Trash', () => {
+
+
+    async function createPost() {
+
+        // Create a Post
+        const title = 'Test Title';
+        await createNewPost( { title } );
+        await publishPost();
+
+    }
+
+
+    afterEach( async () => {
+        await trashAllPosts();
+    } );
+
+    it( 'Empty Trash', async () => {
+
+        await createPost();
+       
+        await visitAdminPage( '/edit.php' );  
+
+        // Move post to trash
+        await page.waitForSelector( '#the-list .type-post' );
+        await page.hover('.row-title');
+        await page.click("a[aria-label='Move “Test Title” to the Trash']");
+
+        // Empty trash 
+        await page.click("a[href='edit.php?post_status=trash&post_type=post']");  
```

Comment:
I think we missed [this](https://github.com/WordPress/wordpress-develop/pull/2186#discussion_r815929153) 😅 .

Reply:
This feedback is addressed, some we tried to use xpath as suggested by you but unfortunately using xpath selector is leading to failing CI, in my local it is working fine but in CI its failing.

Reply:
Why is it failing in CI? Do you have the error messages? We should probably fix that instead of doing it the less preferable way.

Reply:
@kevin940726,

<img width="1244" alt="Screenshot 2022-03-03 at 2 04 08 PM" src="https://user-images.githubusercontent.com/41000648/156527132-88279232-64f3-40a9-abaa-50b301b1d2b9.png">

---